lib/screens/user_registration/widgets/email_feild.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+
+class EmailField extends StatelessWidget {
```

Comment:
That being case what's better practice using a multipurpose TextFormFeild for Institute name and Username which are almost identical in every aspect or using differently?  For the rest mistakes I would rectify them in next pr

Reply:
Actually I wouldn't. As the widget that is in Login makes no sense having ( * ). So only there I could place it however **if we create a property for adding Decorations then it would be best**